### PR TITLE
add editorconfig file

### PR DESCRIPTION
**Pull request by UrNightmaree**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+root = true
+
+[*.lua]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+insert_final_newline = true
```
Comment (waruqi) [inline .editorconfig:9]: why? what's this?

**Comment by waruqi** [comment]
This repository will not consider adding any editor-specific configuration.

**Comment by SirLynix** [comment]
editor-config is pretty standard and widely supported by almost all editors.

https://editorconfig.org/ ("No plugin necessary")

**Comment by UrNightmaree** [comment]
No extension needed for editor-config, it should support VSCode, Lapce, Neovim etc.

**Comment by waruqi** [comment]
Sorry, I don't know about it, but if it is standard and universal, I can consider

**Pull request merged**